client/my-sites/email/email-providers-comparison/index.jsx

```diff
@@ -155,7 +158,7 @@ class EmailProvidersComparison extends React.Component {
 				) }
 				image={ { path: titanLogo } }
 				features={ [
-					translate( 'Monthly billing' ),
+					translate( 'Annual or monthly billing' ),
```

Comment:
Is it worth flagging this for translation now? I am worried this won't get translated by Monday.